util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/ReusableOrmSetupHolder.java

```diff
@@ -619,7 +620,7 @@ private static class DataClearConfigImpl implements DataClearConfig {
 
 		private final List<Class<?>> entityClearOrder = new ArrayList<>();
 
-		private final List<Consumer<Session>> preClear = new ArrayList<>();
+		private final List<ThrowingConsumer<Session, ?>> preClear = new ArrayList<>();
```

Comment:
Ah, I think this is the problem, and the reason you had to turn a `catch (RuntimeException` into a `catch (Throwable` above.

If you apply this suggestion, you'll be able to revert the other changes that add `Throwable` everywhere in this file.

```suggestion
		private final List<ThrowingConsumer<Session, RuntimeException>> preClear = new ArrayList<>();
```

Reply:
yup 😄  that's the culprit 😄. sorry for not replying sooner. I think I've originally done it like you are suggesting here, but then the changes started to span even further, but it might've been in a different place, so I'll double-check once I get a chance.